### ISSUE TITLE
[1.3.0b1] Reverb not working on previously saved projects

### ISSUE THREAD

**Issue by jasll** [polyend/TrackerIssues#368]
I upgraded from 1.2.1 direct to 1.3.0b1 and lost the ability to hear / use reverb in projects I had previously created.

After downgrading and upgrading through firmwares to 1.3.0b1 I continued to have the problem so I opened a demo project (factory content) and reverb worked on projects there. 

I noticed on exiting my own project that the name did not show correctly and was changed from 'fluffy dress' to 03b with a white cursor block. The project seemed corrupted.

I fired up a new project and now reverb works there as it should.

In conclusion it appears there's a chance this latest firmware upgrade could affect personal projects created on firmware 1.2.1. I can't say if this applies to those upgrading from the official 1.2.2 release.

**Comment by ambv** [comment]
This has been fixed in 1.3.0 final. Thanks for reporting.
